api/src/main/java/com/codeforcommunity/rest/subrouter/TeamsRouter.java

```diff
@@ -22,12 +26,71 @@ public TeamsRouter(ITeamsProcessor processor) {
   @Override
   public Router initializeRouter(Vertx vertx) {
     Router router = Router.router(vertx);
-
+    registerCreateTeam(router);
+    registerGetTeam(router);
+    registerAddGoal(router);
+    registerDeleteGoal(router);
     registerDisbandTeam(router);
 
     return router;
   }
 
+  // Create a Team
```

Comment:
Whenever possible try using java doc for comments above classes, methods, or fields. Java doc is just a comment in the form of `/** (whatever you want to say) */`. This doesn't really matter too much for us since we're not going to be generating documentation from our code but it is a good habit to get into